### DISCUSSION TITLE
Remove slayer "Approximate coins spent"

### DISCUSSION THREAD

**Discussion by dukio** [SkyCryptWebsite/SkyCrypt#486] [General]
![image](https://user-images.githubusercontent.com/2744227/120894475-d3ceeb00-c618-11eb-9890-0a1892198f7a.png)

I think this tooltip should be removed because it was introduced before mayor and is now outdated:
- There's Aatrox Mayor who can make slayer quests 50% cheaper
- Since 0.11.5 patch (enderman slayer) the cost of starting slayer quests has changed

These changes make it impossible for us to calculate the real cost grinding those slayers had... I wouldn't even call it approximate anymore.

**Comment by MartinNemi03** [comment]
I agree, but this feature could possibly be brought back with settings where you could set the specific criteria of the calculations. 
Maybe I'm going to ahead with the settings stuff, but overall, I agree.
